src/fs.rs

```diff
@@ -159,8 +159,8 @@ where
             Node::Bytes(b) => (Entry::File(Typ::Bytes, b), Option::None),
             Node::String(t, s) => (Entry::File(t, s.into_bytes()), Option::None),
             Node::List(vs) => {
-                let mut children = HashMap::new();
-                children.reserve(vs.len());
+                let mut children = BTreeMap::new();
+                // children.reserve(vs.len()); TODO (nad) 2023-04-04 deal with BTree and capacity
```

Comment:
There's no way to reserve space in a tree---just not how it works. NBD though, you can drop all of these lines.